### ISSUE TITLE
Some issues in training with different values of max_mel_frames and batch_size

### ISSUE THREAD

**Issue by statcom** [Rayhane-mamah/Tacotron-2#305]
First of all, thanks for sharing this project. Running this has been my hobby since day 1 I found it in Github.

With a machine with AMD TR-1920 and 2 GTX 1080s, I was able to train Tacotron-2 with default dataset for reasonably good TTS after just 30K steps. But there were a couple of issues I need to ask.

* Train/eval loss decreases extremely slow with max_mel_frames=1000 and batch_size=32 (e.g., 0.83 and 0.86 avg loss (train and eval) ) at 30K step). But  avg loss with max_mel_frames=500 and batch_size=64 was 0.64 and 0.63 for train and eval. at just 10K step. Is this expected behavior of training?

* Training speed were 3.1 sec/step with max_mel_frames=1000 and batch_size=32 while 3.8 sec/step with  with max_mel_frames=500 and batch_size=64. But I read others achieved the time less than 1 sec/step. As I didn't change any hyperparameter except for number of GPUs, can anyone tell me how I can achieve the usual training speed? I know 0.75 sec/step can be achieved with Titan XP, but I doubt Titan XP is 3X faster than GTX 1080.

**Comment by statcom** [comment]
OK. I think I understand better now after looking at the code in this repo. Current code does not seem to support Multi-GPU to run for a training task. If one tries, it will actually get slower like what I described before.  But interestingly, I can run two separate training tasks (e.g., English and Korean) with two GPUs. Also setting split_on_cpu-False made each training faster (e.g., 1.75 sec/step).  I am closing this issue.
